lib/pure/asyncdispatch.nim

```diff
@@ -123,15 +123,42 @@
 ##   if future.failed:
 ##     # Handle exception
 ##
-##
 ## Discarding futures
 ## ==================
 ##
-## Futures should **never** be discarded. This is because they may contain
-## errors. If you do not care for the result of a Future then you should
-## use the `asyncCheck` procedure instead of the `discard` keyword. Note
-## however that this does not wait for completion, and you should use
-## `waitFor` for that purpose.
+## Futures should **never** be discarded directly because they may contain
+## errors. If you do not care for the result of a Future then you should use
+## the `asyncCheck` procedure instead of the `discard` keyword. Note that this
+## does not wait for completion, and you should use `waitFor` for that purpose.
```

Comment:
Consider something along the lines of "instead of discarding, use one of the operations mentioned in Handling futures"